profiles/integration-test/config.properties

```diff
@@ -0,0 +1,8 @@
+profile=integration-test
+project.version=1.0-SNAPSHOT
+project.url=https://github.com/reiniergs/polyfill-service
+
+path.polyfillsDist=./polyfills/__dist
+path.browserBaselines=${path.polyfillsDist}/browserBaselines.json
+path.browserAliases=${path.polyfillsDist}/browserAliases.json
+path.featureAliases=${path.polyfillsDist}/aliases.json
```

Comment:
Good question. Probably integration's will override dev's. I can run some tests and check.